test/pixabay-spec.js

```diff
@@ -1,19 +1,19 @@
 'use strict';
 
-var chai = require('chai');
-var chaiAsPromised = require('chai-as-promised');
-var pixabay = require('../src/index');
-var ResultList = require('../src/result-list');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const pixabay = require('../src/index');
+const ResultList = require('../src/result-list');
+const {expect} = chai;
```

Comment:
I still like how 

```javascript
const {expect} = require('chai');
```

looks. Even though we can't use it here, I'll probably be using it in the future.